### PR TITLE
Send view updates

### PR DESCRIPTION
**Pull request by regnerjr**
Fixed bug where you could join a channel named `""`

Also started adding a picker view to the Send VC so that you can choose a channel to send your message to.

### CODE REVIEW

File: Connnect/JoinViewController.swift [hunk 1]
```diff
@@ -16,6 +16,9 @@ class JoinViewController: UIViewController {
     @IBAction func newChannel(sender: UIButton) {
         guard let newChannelName = joinTextField.text else { return }
 
+        guard newChannelName != ""
+            else { return }
+
```
Comment (mysysadmin) [inline Connnect/JoinViewController.swift:21]: Would be good to inform the user inside the place holder text or somewhere else about the mistake

Comment (stellasu168) [reply Connnect/JoinViewController.swift:21]: Like a popup message? Thanks.

Comment (regnerjr) [reply Connnect/JoinViewController.swift:21]: It's pretty clear in use that if you try to create a new channel without a name. Then the button does not do anything

We could do an observer on the text box and only enable the button once they have entered some text.

Comment (mysysadmin) [reply Connnect/JoinViewController.swift:21]: Sounds good

**Review by stellasu168 — APPROVED** [review]

File: Connnect/JoinViewController.swift (hunk 1 above)
Comment (stellasu168) [reply Connnect/JoinViewController.swift:21]: Approved! Should I merge it?

**Pull request merged**